### PR TITLE
duti: init at 1.5.4pre

### PR DESCRIPTION
**Pull request by matthewbauer**
I would init at 1.5.4, but sadly it doesn’t work on OS X 10.11+.

### CODE REVIEW

File: pkgs/os-specific/darwin/duti/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    rev = "7dbcae86f99fedef5a6c4311f032a0f1ca0539cc";
+    sha256 = "1z9sa0yk87vs57d5338y6lvm1v1vvynxb7dy1x5aqzkcr0imhljl";
+  };
+  nativeBuildInputs = [autoreconfHook];
+  buildInputs = [darwin.apple_sdk.frameworks.ApplicationServices];
+  configureFlags = ["--with-macosx-sdk=/homeless-shelter"];
+  meta = with lib; {
+    description = "A command-line tool to select default applications for document types and URL schemes on Mac OS X";
+    longDescription = ''
+duti is a command-line utility capable of setting default applications for
```
Comment (Mic92) [inline pkgs/os-specific/darwin/duti/default.nix]: indentation.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -19145,4 +19145,6 @@ with pkgs;
   undaemonize = callPackage ../tools/system/undaemonize {};
 
   houdini = callPackage ../applications/misc/houdini {};
+
+  duti = callPackage ../os-specific/darwin/duti {};
```
Comment (Mic92) [inline pkgs/top-level/all-packages.nix]: Do we gave up on the alphabetic ordering?

Comment (matthewbauer) [inline pkgs/top-level/all-packages.nix]: Actually should it be in the "Darwin" namespace? Not sure if that's only supposed to be core stuff though.

File: pkgs/os-specific/darwin/duti/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  name = "${pname}-${version}";
+  version = "1.5.4pre";
+  src = fetchFromGitHub {
+    owner = "moretension";
+    repo = pname;
+    rev = "7dbcae86f99fedef5a6c4311f032a0f1ca0539cc";
+    sha256 = "1z9sa0yk87vs57d5338y6lvm1v1vvynxb7dy1x5aqzkcr0imhljl";
+  };
+  nativeBuildInputs = [autoreconfHook];
+  buildInputs = [darwin.apple_sdk.frameworks.ApplicationServices];
```
Comment (LnL7) [inline pkgs/os-specific/darwin/duti/default.nix]: The binary segfaults, we should fix the reference to CoreFoundation until https://github.com/NixOS/nixpkgs/issues/24693 is fixed.
(I suspect that's the issue)

**Comment by LnL7** [comment]
It looks ok after rebasing against may WIP changes.

**Comment by LnL7** [comment]
@matthewbauer The CoreFoundation changes are in master now, should be good to merge now if you rebase.

**Comment by matthewbauer** [comment]
Ok I've rebased and no seg fault!

**Review by LnL7 — APPROVED** [review]

**Pull request merged**